2021-12-13 19:52:58,986 INFO    MainThread:5172 [wandb_setup.py:_flush():71] setting env: {}
2021-12-13 19:52:58,986 INFO    MainThread:5172 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-13 19:52:58,986 INFO    MainThread:5172 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\Usuario\Desktop\Uni\DataDaniMAPSIV\PSIV\wandb\run-20211213_195258-3w4buf7m\logs\debug.log
2021-12-13 19:52:58,986 INFO    MainThread:5172 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\Usuario\Desktop\Uni\DataDaniMAPSIV\PSIV\wandb\run-20211213_195258-3w4buf7m\logs\debug-internal.log
2021-12-13 19:52:58,987 INFO    MainThread:5172 [wandb_init.py:init():404] calling init triggers
2021-12-13 19:52:58,987 INFO    MainThread:5172 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-12-13 19:52:58,989 INFO    MainThread:5172 [wandb_init.py:init():449] starting backend
2021-12-13 19:52:58,990 INFO    MainThread:5172 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn, using: spawn
2021-12-13 19:52:58,993 INFO    MainThread:5172 [backend.py:ensure_launched():199] starting backend process...
2021-12-13 19:52:59,076 INFO    MainThread:5172 [backend.py:ensure_launched():204] started backend process with pid: 14980
2021-12-13 19:52:59,076 INFO    MainThread:5172 [wandb_init.py:init():458] backend started and connected
2021-12-13 19:52:59,078 INFO    MainThread:5172 [wandb_init.py:init():519] updated telemetry
2021-12-13 19:52:59,099 INFO    MainThread:5172 [wandb_init.py:init():549] communicating current version
2021-12-13 19:52:59,643 INFO    MainThread:5172 [wandb_init.py:init():554] got version response 
2021-12-13 19:52:59,643 INFO    MainThread:5172 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-13 19:52:59,881 INFO    MainThread:5172 [wandb_init.py:init():592] starting run threads in backend
2021-12-13 19:52:59,981 INFO    MainThread:5172 [wandb_run.py:_console_start():1816] atexit reg
2021-12-13 19:52:59,983 INFO    MainThread:5172 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.WRAP
2021-12-13 19:52:59,983 INFO    MainThread:5172 [wandb_run.py:_redirect():1727] Wrapping output streams.
2021-12-13 19:52:59,986 INFO    MainThread:5172 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-13 19:52:59,986 INFO    MainThread:5172 [wandb_init.py:init():619] run started, returning control to user process
2021-12-13 19:52:59,987 INFO    MainThread:5172 [wandb_config.py:__setitem__():144] config set learning_rate = 0.001 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000015FC8AD5AC0>>
2021-12-13 19:52:59,987 INFO    MainThread:5172 [wandb_run.py:_config_callback():962] config_cb learning_rate 0.001 None
2021-12-13 19:52:59,987 INFO    MainThread:5172 [wandb_config.py:__setitem__():144] config set batch_size = 32 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000015FC8AD5AC0>>
2021-12-13 19:52:59,987 INFO    MainThread:5172 [wandb_run.py:_config_callback():962] config_cb batch_size 32 None
2021-12-13 19:53:01,666 INFO    MainThread:5172 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-12-13 19:53:01,667 INFO    MainThread:5172 [wandb_run.py:_restore():1758] restore
2021-12-13 19:53:02,058 INFO    MainThread:5172 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2021-12-13 19:53:02,363 INFO    MainThread:5172 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2021-12-13 19:53:02,922 INFO    MainThread:5172 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 4221
}

2021-12-13 19:53:03,025 INFO    MainThread:5172 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 4221
}

2021-12-13 19:53:03,135 INFO    MainThread:5172 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 4221
}

2021-12-13 19:53:03,245 INFO    MainThread:5172 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4221
  total_bytes: 4221
}

2021-12-13 19:53:03,355 INFO    MainThread:5172 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4221
  total_bytes: 4221
}

2021-12-13 19:53:03,466 INFO    MainThread:5172 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4221
  total_bytes: 4221
}

2021-12-13 19:53:03,576 INFO    MainThread:5172 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4221
  total_bytes: 4221
}

2021-12-13 19:53:03,685 INFO    MainThread:5172 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4221
  total_bytes: 4221
}

2021-12-13 19:53:03,794 INFO    MainThread:5172 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4221
  total_bytes: 4221
}

2021-12-13 19:53:04,190 INFO    MainThread:5172 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4221
  total_bytes: 4221
}

2021-12-13 19:53:04,711 INFO    MainThread:5172 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4221
  total_bytes: 4221
}
local_info {
}

2021-12-13 19:53:05,775 INFO    MainThread:5172 [wandb_run.py:_append_files():2184] logging synced files
